src/layers/conv.jl

```diff
@@ -343,13 +381,17 @@ end
 """
     CrossCor(filter, in => out, σ=identity; stride=1, pad=0, dilation=1, [bias, init])
 
-Standard cross convolutional layer. `filter` is a tuple of integers
+Standard cross correlation layer. `filter` is a tuple of integers
 specifying the size of the convolutional kernel;
 `in` and `out` specify the number of input and output channels.
 
 Parameters are controlled by additional keywords, with defaults
 `init=glorot_uniform` and `bias=true`.
 
+CrossCor layer can also be manually constructed by passing in weights and
+biases. This constructor accepts the layer accepts the same keywords (and has
+the same defaults) as the `CrossCor((4,4), 3 => 7, relu)` method.
```

Comment:
```suggestion
the same defaults) as `CrossCor(k::NTuple{N,Integer}, ch::Pair{<:Integer,<:Integer}, σ; ...)`.
```
Better to use the signature so that users know which constructor to look for.